lib/pages/settings/view/feedback_form.dart

```diff
@@ -0,0 +1,166 @@
+import 'package:acs_upb_mobile/generated/l10n.dart';
+import 'package:acs_upb_mobile/pages/settings/model/issue.dart';
+import 'package:acs_upb_mobile/pages/settings/service/issue_provider.dart';
+import 'package:acs_upb_mobile/widgets/scaffold.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_feather_icons/flutter_feather_icons.dart';
+import 'package:provider/provider.dart';
+import 'package:acs_upb_mobile/resources/theme.dart';
+
+class FeedbackFormPage extends StatefulWidget {
+  static const String routeName = '/feedbackForm';
+
+  @override
+  _FeedbackFormPageState createState() => _FeedbackFormPageState();
+}
+
+class _FeedbackFormPageState extends State<FeedbackFormPage> {
+  final _formKey = GlobalKey<FormState>();
+  TextEditingController issueEmailController = TextEditingController();
+  TextEditingController issueController = TextEditingController();
+  bool _issueSelected = false;
+  bool _feedbackSelected = true;
+
+  @override
+  Widget build(BuildContext context) {
+    final issueProvider = Provider.of<IssueProvider>(context);
+
+    return Form(
+        key: _formKey,
+        child: AppScaffold(
+          title: Text(S.current.labelFeedback),
+          actions: [
+            AppScaffoldAction(
+              text: S.current.buttonSubmit,
+              onPressed: () {
+                // Validate returns true if the form is valid, or false otherwise.
+                if (_formKey.currentState.validate()) {
+                  // If the form is valid, it displays a snackbar and saves the
+                  // information in the cloud
+                  ScaffoldMessenger.of(context).showSnackBar(
```

Comment:
Is it bad practice if I just add a toast in the try-catch for the makeIssue function, 1 toast in the try that shows "Feedback sent", and then replace the "Something went wrong" toast in the catch with "Feedback could not be sent"? 

Regarding not popping the page after the feedback is submitted, I tried it but couldn't make it work, so I left it like that

Reply:
Yeah, it's bad practice because then you have one toast from the provider but another from the UI, which can lead to unexpected behaviour. It's better if only one side deals with the toasts for a given call, and I think the view is better suited for this than the provider.

Reply:
Also, what exactly didn't work with popping? You can't just not do something because it didn't work xD